src/logos/vsp.svg

```diff
@@ -2,7 +2,7 @@
   xmlns="http://www.w3.org/2000/svg"
   width="72.183"
   height="79"
-  viewbox="0 0 72.183 79"
+  viewbox="0 0 72.183 72.183"
```

Comment:
`height` needs to be changed too.

Reply:
thanks, I failed to paste that one! here it is https://github.com/vesperfi/metadata/pull/13/commits/1f853ca764039c9d8592d8f3a93686b58f379d2e